code/modules/mob/living/simple_animal/friendly/gondola.dm

```diff
@@ -25,6 +25,8 @@
 	maxHealth = 200
 	health = 200
 	del_on_death = TRUE
+	var/mob/living/carbon/human/stored_mob //could be changed
+	var/M
```

Comment:
?
```suggestion
```

Reply:
revert yarn

Reply:
Okay, i will revert yarn, but if i will remove var/M gondola touch wouldn’t work.

Reply:
it would work